go.mod

```diff
@@ -34,7 +34,7 @@ require (
 	github.com/docker/go-units v0.3.3 // indirect
 	github.com/docker/libtrust v0.0.0-20160708172513-aabc10ec26b7 // indirect
 	github.com/dsnet/compress v0.0.1 // indirect
-	github.com/fatih/color v1.7.0 // indirect
+	github.com/fatih/color v1.7.1-0.20181010231311-3f9d52f7176a
```

Comment:
Is there a reason you're using a specific commit of this package instead of using a tagged version?

Reply:
Umm, I guess this vendor was added after I made my PR (it was not there a couple weeks ago)... I'll change it back.